eip/EIPS/eip-4337.md

```diff
@@ -624,6 +624,219 @@ Returns an array of the entryPoint addresses supported by the client. The first
 }
 ```
 
+#### * eth_chainId
+
+Returns EIP155 Chain ID.
+
+```json=
+# Request
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "eth_chainId",
+  "params": []
+}
+
+# Response
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": "0x1"
+}
+```
+
+### debug Namespace
+
+#### * debug_bundler_clearState
+
+Clears the bundler mempool and reputation data of paymasters/accounts/factories/aggregators.
+
+```json=
+# Request
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "debug_bundler_clearState",
+  "params": []
+}
+
+# Response
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": "ok"
+}
+```
+
+#### * debug_bundler_dumpMempool
+
+Returns:
+
+`array` - Array of mempool entries objects, or an empty array.
+
+where each mempool entry has the following fields:
+
+`userOp` - UserOperation object. See `eth_sendUserOperation` for more details.
+`prefund` - The calculated prefund required for the user operation execution.
```

Comment:
these are implementation-specific fields. I think the "dump" should only dump the raw UserOp entries

Reply:
Then don't return it in the bundler reference implementation... I went by the code.